tests/shear_heating_with_melt/screen-output

```diff
@@ -97,17 +96,17 @@ Number of degrees of freedom: 1,663 (486+202+486+82+243+82+82)
    Rebuilding Stokes preconditioner...
    Solving Stokes system... 0+0 iterations.
    Solving fluid velocity system... 1 iterations.
-      Relative nonlinear residuals (temperature, compositional fields, Stokes system): 4.91022e-07, 9.24119e-05, 3.1096e-07, 1.60801e-08
-      Relative nonlinear residual (total system) after nonlinear iteration 3: 9.24119e-05
+      Relative nonlinear residuals (temperature, compositional fields, Stokes system): 4.91022e-07, 9.2412e-05, 3.1096e-07, 1.60806e-08
```

Comment:
tiny differences can be ignored, I think. (I will try to remove the changes and see if the test still passes)

Reply:
it does.